gammapy/morphology/overlap.py

```diff
@@ -175,7 +175,7 @@ def get_containment_mask(glon_pos, glat_pos, r_containment, shape):
     hdulist = fits.open('../counts.fits')
     w = WCS(hdulist[0].header)
     y, x = np.indices(shape)
-    glon, glat = w.wcs_pix2world(x, y, 1)
+    glon, glat = w.wcs_pix2world(x, y, 0)
```

Comment:
Could you please everywhere change this to explicitly pass `origin=0` via a keyword argument (instead of just `0` as third positional argument)?
I think this makes for more readable code.